doc/source/tutorials/elastic_networks.rst

```diff
@@ -0,0 +1,132 @@
+================
+Elastic Networks
+================
+
+The first method applied to maintain the secondary and tertiary structure
```

Comment:
Perhaps mention here as well why we need them? It is obvious from our point of view but coming from AA this is really nothing you ever think of. Potentially, you also could mention that there are different flavors which may impact dynamics and properties of your protein. The better then structure bias the better the result.